aries_vcx/src/handlers/out_of_band/sender/sender.rs

```diff
@@ -54,33 +54,18 @@ impl OutOfBandSender {
 
     pub fn append_a2a_message(mut self, msg: A2AMessage) -> VcxResult<Self> {
         let (attach_id, attach) = match msg {
-            A2AMessage::CredentialRequest(request) => {
-                (AttachmentId::CredentialRequest,
-                serde_json::to_string(&request)
-                    .map_err(|_| VcxError::from_msg(VcxErrorKind::SerializationError, format!("Failed to serialize request: {:?}", request)))?)
-            }
             A2AMessage::PresentationRequest(request) => {
-                (AttachmentId::PresentationRequest,
-                serde_json::to_string(&request)
-                    .map_err(|_| VcxError::from_msg(VcxErrorKind::SerializationError, format!("Failed to serialize request: {:?}", request)))?)
+                (AttachmentId::PresentationRequest, json!(&request).to_string())
```

Comment:
Simplified the stringification - this was there's no possibility of an error being throw